stream-parent/stream-client/src/main/java/org/interledger/stream/sender/SimpleStreamSender.java

```diff
@@ -298,6 +347,9 @@ default int totalPackets() {
       this.receiverDenomination = Optional.empty();
 
       this.unrecoverableErrorEncountered = new AtomicBoolean(false);
+
+      Objects.requireNonNull(sendPacketSleep);
+      this.sendPacketSleep = sendPacketSleep.orElse(UnsignedLong.valueOf(100L)).longValue();
```

Comment:
There was talk that we should make this default much lower, like 10 -- @nhartner @theotherian does that sound OK?

Reply:
I set the default to 10ms.  After running some tests, there doesn't seem to be a difference in total send time between `SimpleStreamSender`s with a 10ms sleep and 5ms sleep.

We should maybe do some more testing to figure out the most optimal sleep time, but that's probably overkill for now

Reply:
Per the Quilt community call - let's explore 20ms or 50ms and just double-check that 10ms seems to be the best sleep time.

Reply:
Ran a bunch of different sleep times.  10ms always seems to execute faster than anything > 10ms.  Once you get below 10ms, it's a wash which one will be faster.

Note this was tested on a payment that sends roughly 20 packets.